### ISSUE TITLE
Rename to `viam-dart-sdk`?

### ISSUE THREAD

**Issue by clintpurser** [viamrobotics/viam-flutter-sdk#1]
Not sure how else to ask this right now, so i am creating an issue!

I am wondering if this really is a `flutter` SDK, or if it is more like a dart package / SDK. Perhaps naming it `viam-dart-sdk` would be more representative of what it really is?

Obviously I am still looking at this from the outside and I am probably missing details and a lot of context. This was just a thought that came to mind.

Thoughts @njooma ?

**Comment by njooma** [comment]
Reasonable question! Originally I had named this `viam-dart-sdk` because you are correct, it's more of a dart package. However, this iteration of the package has some dependencies that depend on flutter (namely `auth0-flutter`). 

Eventually, I would love to have a separate `dart-sdk` that specifically handles all the dart code, and then a `flutter-sdk` that includes the `dart-sdk`, and also adds in flutter specific functionality (auth0 integration, additional UI for logging in maybe?, etc.). 

But for right now, since this SDK still has dependencies that require on flutter, I opted to call it `flutter-sdk`
